service/src/main/java/bio/terra/workspace/app/configuration/external/StairwayDatabaseConfiguration.java

```diff
@@ -1,14 +1,14 @@
 package bio.terra.workspace.app.configuration.external;
 
-import bio.terra.workspace.app.configuration.BaseDatabaseConfiguration;
+import bio.terra.common.db.BaseDatabaseProperties;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 @EnableConfigurationProperties
 @ConfigurationProperties(prefix = "workspace.stairway-database")
-public class StairwayDatabaseConfiguration extends BaseDatabaseConfiguration {
```

Comment:
These are the only two classes using `BaseDatabaseConfiguration`, delete it?

Reply:
Yes, will do!

Reply:
Wait, are you saying two uses isn't enough to warrant a base class?

Reply:
Not at all. Instead of using the WSM base class, this changes to using the Terra Common Lib base class. So the sharing is more not less.

The issue was that when I scanned the TCL `db` package, it created conflicts with the WSM base class. By using the TCL base class, I remove that conflict and am able to have a database config contained in my TPS library.